ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -722,16 +692,32 @@ let get_group_key ~__context ~vm =
   | Some x -> x
   | None -> `Other
 
-let group_hosts_by_best_pgpu ~__context vgpu =
-  let vgpu_type = Db.VGPU.get_type ~__context ~self:vgpu in
-  let gpu_group = Db.VGPU.get_GPU_group ~__context ~self:vgpu in
-  match
-    Xapi_gpu_group.get_remaining_capacity_internal ~__context
-      ~self:gpu_group ~vgpu_type
-  with
-  | Either.Left e -> raise e
-  | Either.Right _ -> ();
-    group_hosts_by_best_pgpu_in_group ~__context gpu_group vgpu_type
+(* Rank the provided hosts (visible_hosts) by the remainding capacity for the
```

Comment:
Mostly grammar:
Rank the provided hosts by their remaining capacity for the given vGPU. The vGPU has an associated vGPU group. We get the rank preference from the vGPU group.